sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -81,32 +85,43 @@ export class EducationProgramOfferingController {
     return createdProgramOffering.id;
   }
 
+  /**
+   * Offering Summary for an institution location
```

Comment:
LocationId missing in the comment @param

Reply:
thanks for the catch 👍